instrumentation/servlet/servlet-3.0/javaagent-tests/src/test/java/io/opentelemetry/javaagent/instrumentation/servlet/v3_0/mapping/TomcatServlet3MappingTest.java

```diff
@@ -0,0 +1,81 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.javaagent.instrumentation.servlet.v3_0.mapping;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.util.UUID;
+import javax.servlet.Servlet;
+import org.apache.catalina.Context;
+import org.apache.catalina.LifecycleException;
+import org.apache.catalina.startup.Tomcat;
+
+class TomcatServlet3MappingTest extends AbstractServlet3MappingTest<Tomcat, Context> {
+  @Override
+  protected Tomcat setupServer() throws Exception {
+    Tomcat tomcatServer = new Tomcat();
+
+    File baseDir;
+    try {
+      baseDir = Files.createTempDirectory("tomcat").toFile();
+    } catch (IOException e) {
+      throw new RuntimeException(e);
+    }
```

Comment:
since the method already has `throws Exception` I think you can skip catching the exception